pkg/sshd/server.go

```diff
@@ -49,37 +107,24 @@ const (
 // This puts the following variables into the context:
 // 	- ssh.Closer (chan interface{}): Send a message to this to shutdown the server.
```

Comment:
@smothiki the documentation about what will go into the context is no longer valid. can you remove it?